README.md

```diff
@@ -113,6 +100,7 @@ Using `v-model`
 | fullscreenMobile | Boolean              | false            | Datepicker will appear in a bottom sheet in mobile view                                                                                                                                                                 |
 | minDate          | String\|Number\|Date |                  | Date are available FROM this date                                                                                                                                                                                       |
 | maxDate          | String\|Number\|Date |                  | Date are disabled AFTER this date                                                                                                                                                                                       |
+| yearRangePicker  | Number               | 10               | Year picker range                                                                                                                                                                                                       |
```

Comment:
visibleYears or visibleYearsNumber would be more appropriate.

Reply:
yes, bit struggling in choosing appropriate name >.<